scripts/dev/count_lines_of_code

```diff
@@ -140,15 +141,15 @@ def IsDirExcluded(path):
 
    return False
 
-def IsFileExcluded(fname):
+def IsFileExcluded(fname: str) -> bool:
    return fname in excludedFiles
 
-def AccumulateCounters(destArr, arr):
+def AccumulateCounters(destArr: Dict[str, int], arr: Dict[str, int]):
 
    for ft in fileTypes:
       destArr[ft] += arr[ft]
 
-def PrintRow(cols):
+def PrintRow(cols: List[str]) -> None:
```

Comment:
`PrintRow()` was supposed to take a `List` of `Any`, not a `List` of `str`, that's why it does `str(val)`. By forcing the type of the list, the `str(val)` because pointless. I'd make this `List[Any]`.

Reply:
Ok, perfect